feature-group-definitions/webp.yml

```
@@ -1,3 +1,3 @@
-spec: https://developers.google.com/speed/webp/
+spec: https://www.ietf.org/archive/id/draft-zern-webp-13.html
```

I haven't seen the https://www.ietf.org/archive/id/draft-zern-webp-13.html style of URL for IETF documents before, but it's a permalink to the currently most recent version, 13. I think https://datatracker.ietf.org/doc/html/draft-zern-webp/ is better and "more nightly" since it's going to reflect version 14 when that is published:

```suggestion
spec: https://datatracker.ietf.org/doc/html/draft-zern-webp/
```

---

In that case, should the URLs from https://github.com/w3c/browser-specs/pull/1120 be flipped around? Or should I expand the set of allowable URLs? If I apply the change from this comment, it'll fail the check.

---

Oh, we only allow nightly URLs, I forgot that. In that case I think browser-specs should change. I've filed https://github.com/w3c/browser-specs/issues/1127 but think we can live with the versioned URL until that is fixed.